plugins/modules/digital_ocean_load_balancer.py

```diff
@@ -41,18 +35,24 @@
       - The available sizes are C(lb-small), C(lb-medium), or C(lb-large).
       - You can resize load balancers after creation up to once per hour.
       - You cannot resize a load balancer within the first hour of its creation.
+      - This field has been replaced by the size_unit field for all regions except in AMS2, NYC2, and SFO1.
```

Comment:
Should we explain here what the user has to do? Something like:

> When creating a load balancer in the `AMS2`, `NYC2`, or `SFO1` regions, you must specify the `size` field. Otherwise, you must specify the `size_unit` field.

Should we also clarify that users can pass in both, to be on the safe side, and the module will figure out which one is needed based on the region?
I guess I'm also not clear on how error checking works. (are we not able to detect a missing field until we go to call the API?)

Reply:
I think we're explaining it well enough in the relevant `description` items, and, I'm hoping this edge case goes away in time.

Reply:
We're making the `size` and `size_unit` parameters mutually-exclusive (Ansible will throw an error if they're both used) like this:

```python
        mutually_exclusive=(
            [
                ("tag", "droplet_ids"),
                ("size", "size_unit"),
            ]
        ),
```

And, we're using one-or-the-other like this:

```python
        if self.region in DOLoadBalancer.size_regions:
            self.module.params.pop("size_unit")
        else:
            self.module.params.pop("size")
```

I suppose we'll need to ensure that we have one-or-the-other after this pop -- I'll add a check for that.